<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RockPaperScissors</class>
 <widget class="QMainWindow" name="RockPaperScissors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
background-color: rgb(16, 16, 16);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Background/window.png);
border: 3px solid black;
</string>
   </property>
   <widget class="QLabel" name="LetsPlay">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>101</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.522, y2:0.619, stop:0.0199005 rgba(255, 255, 255, 0));
	background-image: url(:/images/banner-wave-3.svg);
color:black;
qproperty-alignment: AlignCenter;
border: no border;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Let's Play </string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>14</number>
    </property>
   </widget>
   <widget class="QLabel" name="comp_choice">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>186</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel
{
background-color:white;
	background-image: url(:/images/banner-wave-3.svg);
border-width: 2px;
    border-radius: 5px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="choosen">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel
{
background-color:white;
	background-image: url(:/images/banner-wave-3.svg);
border-width: 2px;
    border-radius: 5px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:black;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 255, 255, 0));
background-image:url(:/images/banner-wave-3.svg);
border: no border;


}</string>
    </property>
    <property name="text">
     <string>System</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <property name="indent">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="rock">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 3px solid #8f8f91;
    border-radius: 6px;
	
	
	border-image: url(:/icons/rock.png);
	background-image: url(:/transparent/banner-wave-3.svg);
	
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="paper">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>90</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 3px solid #8f8f91;
    border-radius: 6px;
	
	border-image: url(:/icons/paper.png);
    
	background-image: url(:/transparent/banner-wave-3.svg);
	
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="scissors">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 3px solid #8f8f91;
    border-radius: 6px;
	
	border-image: url(:/icons/scissors.png);
    background-image: url(:/transparent/banner-wave-3.svg);
	
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Yourname">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:black;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 255, 255, 0));
	background-image: url(:/images/banner-wave-3.svg);
border: no border;

}</string>
    </property>
    <property name="text">
     <string>You</string>
    </property>
    <property name="indent">
     <number>15</number>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>620</y>
      <width>86</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 3px solid #8f8f91;
    border-radius: 6px;
	
	background-image: url(:/transparent/banner-wave-3.svg);
    
	
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}
</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 3px solid #8f8f91;
    border-radius: 6px;
	
	background-image: url(:/transparent/banner-wave-3.svg);
	
	background-color: rgb(0, 0, 0);
color: white;
    
	
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}
</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>520</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: black;
border: 2px solid black;
    border-radius: 4px;
    padding: 2px;
	background-image: url(:/images/banner-wave-3.svg);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.522, y2:0.619, stop:0.0199005 rgba(255, 255, 255, 0));

}</string>
    </property>
    <property name="text">
     <string>System Score</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.522, y2:0.619, stop:0.0199005 rgba(255, 255, 255, 0));
	background-image: url(:/images/banner-wave-3.svg);
border: 2px solid black;
    border-radius: 4px;
    padding: 2px;
color:black;
}</string>
    </property>
    <property name="text">
     <string>Your Score</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="your_score">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>560</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-image: url(:/transparent/banner-wave-3.svg);
background-color:white;
	border: 2px solid black;
    border-radius: 4px;
    padding: 2px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sys_score">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>560</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-image:url(:/transparent/banner-wave-3.svg);
background-color:white;
border: 2px solid black;
    border-radius: 4px;
    padding: 2px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>701</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/transparent/banner-wave-3.svg);
border: no border;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Rresource.qrc"/>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
